builder/builder-next/exporter/wrapper.go

```diff
@@ -52,6 +54,9 @@ func (e *imageExporterMobyWrapper) Resolve(ctx context.Context, id int, exporter
 	if _, has := exporterAttrs[string(exptypes.OptKeyDanglingPrefix)]; !has {
 		exporterAttrs[string(exptypes.OptKeyDanglingPrefix)] = "moby-dangling"
 	}
+	for k, v := range e.extraAttrs {
+		exporterAttrs[k] = v
```

Comment:
This should not overwrite attrs from the user.

(We should really get rid of this wrapper at all as well.)